code/modules/awaymissions/mob_spawn.dm

```diff
@@ -609,8 +616,49 @@
 	icon_state = "sleeper"
 	flavour_text = "Moo!"
 
+///these mob spawn subtypes trigger immediately (New or Initialize) and are not player controlled... since they're dead, you know?
```

Comment:
```suggestion
/// these mob spawn subtypes trigger immediately (New or Initialize) and are not player controlled... since they're dead, you know?
```